### ISSUE TITLE
Unable to open Container Sheet (similar to #246)

### ISSUE THREAD

**Issue by jhathawaytn** [vttred/ose#249]
### What happened?

1) Create a new container an name it.
2) Click on newly added container in side bar, will not open.

### What is the expected behaviour?

When I click on item in side bar, it should open for edit.

### Relevant Errors and Warnings

```shell
Error: An error occurred while rendering OseItemSheet 110. Parse error on line 1:
{{#if {isV10}}}<form clas
------^
Expecting 'CLOSE_RAW_BLOCK', 'CLOSE', 'CLOSE_UNESCAPED', 'OPEN_SEXPR', 'CLOSE_SEXPR', 'ID', 'OPEN_BLOCK_PARAMS', 'STRING', 'NUMBER', 'BOOLEAN', 'UNDEFINED', 'NULL', 'DATA', 'SEP', got 'INVALID'
[Detected 2 packages: lib-wrapper, monks-active-tiles]
    parseError handlebars.min.js:28
    parse handlebars.min.js:28
    d handlebars.min.js:27
    e handlebars.min.js:27
    d handlebars.min.js:28
    e handlebars.min.js:28
    renderTemplate foundry.js:6906
    _renderInner foundry.js:5230
    _renderInner foundry.js:5791
    _render foundry.js:5073
    _render foundry.js:5780
    render foundry.js:5026
    render foundry.js:6352
    _onClickDocumentName foundry.js:60169
    call_wrapped libWrapper-wrapper.js:507
    checkClickDocumentName monks-active-tiles.js:1817
    call_wrapper libWrapper-wrapper.js:616
    _onClickDocumentName#0 libWrapper-wrapper.js:189
    call_wrapped libWrapper-wrapper.js:507
    clickDocumentName monks-active-tiles.js:1765
    call_wrapper libWrapper-wrapper.js:616
    _onClickDocumentName#0 libWrapper-wrapper.js:189
    jQuery 2
foundry.js:747:30
    onError foundry.js:747
    call_wrapped libWrapper-wrapper.js:507
    <anonymous> listeners.js:134
    onError#0 libWrapper-wrapper.js:187
    render foundry.js:5028
    (Async: promise callback)
    render foundry.js:5026
    render foundry.js:6352
    _onClickDocumentName foundry.js:60169
    call_wrapped libWrapper-wrapper.js:507
    checkClickDocumentName monks-active-tiles.js:1817
    call_wrapper libWrapper-wrapper.js:616
```

### Foundry VTT Core Version

V10

### Old-School Essentials Core Version

1.6.6

### Please update your browser regularly. Which (updated) browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jhathawaytn** [comment]
I may have jumped the shark on this one.  I see there is already a fix coming.  Sorry.

**Comment by wyrmisis** [comment]
@jhathawaytn This should be fixed in #248 (and available in release 1.6.7)
